config/database.yml

```diff
@@ -17,9 +17,10 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  # For details on connection pooling, see Rails configuration guide
-  # https://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  pool: 5
+  username: projectmatch
+  password: projectmatchpass
```

Comment:
Again, not a priority right now but we could consider using our container's environment variables to save and access credentials.